website/www/site/content/en/roadmap/python-sdk.md

```diff
@@ -23,7 +23,7 @@ The Apache Beam community voted to join [the pledge](https://python3statement.or
 
 ## Python 3 Support
 
-Apache Beam 2.14.0 and higher support Python 3.5, 3.6, and 3.7. We're continuing to [improve](https://issues.apache.org/jira/browse/BEAM-1251?focusedCommentId=16890504&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1689050) the experience for Python 3 users and add support for Python 3.x minor versions ([BEAM-8494](https://issues.apache.org/jira/browse/BEAM-8494)):
+Apache Beam 2.23.0 and higher support Python 3.5, 3.6, 3.7, and 3.8. We're continuing to [improve](https://issues.apache.org/jira/browse/BEAM-1251?focusedCommentId=16890504&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1689050) the experience for Python 3 users and add support for Python 3.x minor versions ([BEAM-8494](https://issues.apache.org/jira/browse/BEAM-8494)):
```

Comment:
I think it's worth keeping the first version at which Python 3 support was added. Also, 3.5 support has now been dropped (as has Python 2.x).

Reply:
Thanks! I would combine the sections into something like:

## Adding support for new Python versions and sunsetting older ones.

Beam aims to align support of Python versions
with Python annual release cycle (https://www.python.org/dev/peps/pep-0602/). This means: 

1. Beam will aim to add support for a new Python 3.x version after
the version is released by Python community.
2. After a Python 3.x version reaches the end of support (https://www.python.org/downloads/), Beam will remove
support for this version in Beam, starting from the first Beam release that
is cut after the end-of-support date.
3. The rules above are our default course of action, but can be adjusted on
a case-by-case basis via a discussion on dev@.

Adding support of a new version of Python requires updating test infrastructure
to run tests against the new version, making sure Beam's dependencies support
new Python version, and verifying Beam runners support Python 3.X. Scoping
of tasks can be coordinated via Jira and dev@ mailing list if necessary.

(this can be made into a table perhaps):

Apache Beam 2.13.0 added support for Python 3.5.
Apache Beam 2.14.0 added support Python 3.5, 3.6, and 3.7. 
Apache Beam 2.23.0 added support for Python 3.8,
Apache Beam 2.25.0 removed support for Python 2 and Python 3.5.

Tracking issue for Python 3.8: https://issues.apache.org/jira/browse/BEAM-8494
Tracking issue for Python 3.9: https://issues.apache.org/jira/browse/BEAM-12000

Contributions are welcome!
...

Reply:
@zdog234 feel free to massage the message above into the file or i can send a follow up commit on this branch.